tests/Acoustics.Test/AnalysisPrograms/Recognizers/GenericRecognizerTests.cs

```diff
@@ -1204,5 +1208,83 @@ public void TestMultipleAlgorithms()
             Assert.AreEqual("DTMFupper", @event.Name);
             Assert.AreEqual("UpperBandDTMF_z", @event.Profile);
         }
+
+        [TestMethod]
+        public void TestOscillationPlusHarmonics()
+        {
+            var config = new GenericRecognizer.GenericRecognizerConfig()
+            {
+                Profiles = new Dictionary<string, object>()
+                {
+                    {
+                        "TestBlob", new BlobParameters()
+                        {
+                            FrameSize = 1024,
+                            FrameStep = 1024,
+                            MaxHertz = 7200,
+                            MinHertz = 4800,
+                            BgNoiseThreshold = 0.0,
+                            BottomHertzBuffer = 1000,
+                            TopHertzBuffer = 500,
+                            DecibelThresholds = new double?[] { 3.0 },
+                        }
+                    },
+                    {
+                        "LowerBandDTMF_z",
```

Comment:
the name of the test method and test profile do not match up.

DTMF tones are harmonic not oscillations. 

Probably not wrong but it is confusing